src/test/suite/format/samples/ahk-explorer.out.ahk

```diff
@@ -3493,25 +3493,25 @@ return
 #if winactive("renamingWinTitle ahk_class AutoHotkeyGUI")
 
 $esc::
-    if (focused="flistView") {
-        if (canRename) {
-            canRename:=false
-            ; gui, renameSimple:Default
-            ; gui, submit
-            gui, main:Default
-            ControlFocus,, % "ahk_id " ListviewHwnd%whichSide%
+if (focused="flistView") {
```

Comment:
Do we know why we are no longer indenting after labels? Is this intentional? I understand that all code under a label is executed until a `return` statement is hit, but generally each label has its own return statement.

I'm not sure users would appreciate this change, but I may be wrong. Do you have code samples that show code is usually not indented after a label?

My personal preference would be to make all labels have no indent, but code under them (up to and including return statements) has one level of indentation.

```
$esc::
    doThing()
^a::
    doAnotherThing()
    return

functionCallOutsideLabel()

#x::
    lastThing()
    return
moreCodeHere()
```

Reply:
> Do we know why we are no longer indenting after labels?
No.
> Is this intentional?
No.